Helm/opensearch/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,12 @@
+---
+{{- if .Values.maxUnavailable }}
+apiVersion: policy/v1beta1
```

Comment:
Deprecated to since v1.21 and will be removed in v1.25

https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125

Reply:
Noted !!!